pubsub/topics.js

```diff
@@ -0,0 +1,181 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START auth]
+// By default, gcloud will authenticate using the service account file specified
+// by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use the
+// project specified by the GCLOUD_PROJECT environment variable. See
+// https://googlecloudplatform.github.io/gcloud-node/#/docs/guides/authentication
+var gcloud = require('gcloud');
+
+// Get a reference to the pubsub component
+var pubsub = gcloud.pubsub();
+// [END auth]
+
+// [START create_topic]
+/**
+ * Create a new topic.
+ *
+ * @param {string} topicName Name for the new topic.
+ * @param {Function} callback The callback function.
+ */
+function createTopic (topicName, callback) {
+  if (!topicName) {
+    return callback(new Error('"topicName" is required!'));
+  }
+
+  var topic = pubsub.topic(topicName);
+
+  // Get the topic if it exists. Create it if it does not exist.
+  topic.get({
```

Comment:
Is this the most idiomatic way to do this? Is there not just a 'topic.create()`?